lib/loader/index.js

```diff
@@ -135,6 +186,16 @@ module.exports = {
       var firstScript = getFirstScript();
 
       firstScript.parentNode.insertBefore(script, firstScript);
+
+      _loadedUrls[url] = true;
+
+      if (options.namespaceName) {
+        namespace.loadedUrls = _loadedUrls;
```

Comment:
They can de-reference it or assign properties of the property. `registerProperty` is just a helper that throws an error if it happens to exist already. So if you are performing possible destructive ops you can use it to be safe.